Topics/Development_Process/Technical_Documents/Intro_to_rfcs.md

```diff
@@ -0,0 +1,96 @@
+# **Introduction to RFCs**
+
+The following is an introductory overview of RFCs based on the following 2 articles by Juan Pablo Buriticá:
+
+1. ["A thorough team guide to RFCs"](https://leaddev.com/technical-decision-making/thorough-team-guide-rfcs)
+2. ["6 Lessons I learned while implementing technical RFCs as a decision making tool"](https://buriti.ca/6-lessons-i-learned-while-implementing-technical-rfcs-as-a-management-tool-34687dbf46cb)
+
+## **What are RFCs?**
+
+- RFC stands for Request for Comment
```

Comment:
```suggestion
- RFC stands for Request for Comments
```

The first point uses 'Comment' while the 5th point uses 'comments'. It would be better to change one to match the other for consistency.

Reply:
Nice catch! That's a great point